source/operator.rst

```diff
@@ -6,832 +6,111 @@ The operator is configured via the spec section of the
-|                                |           |          | ration/sec |
-|                                |           |          | ret/#using |
-|                                |           |          | -imagepull |
-|                                |           |          | secrets>`_ |
-|                                |           |          | _          |
-|                                |           |          | for the    |
-|                                |           |          | Percona    |
-|                                |           |          | XtraDB     |
-|                                |           |          | Cluster    |
-|                                |           |          | docker     |
-|                                |           |          | image      |
-+--------------------------------+-----------+----------+------------+
-| labels                         | label     | ``rack:  | The        |
-|                                |           | rack-22` | `Kubernete |
-|                                |           | `        | s          |
-|                                |           |          | affinity   |
-|                                |           |          | labels <ht |
-|                                |           |          | tps://kube |
-|                                |           |          | rnetes.io/ |
-|                                |           |          | docs/conce |
-|                                |           |          | pts/config |
-|                                |           |          | uration/as |
-|                                |           |          | sign-pod-n |
-|                                |           |          | ode/>`__   |
-+--------------------------------+-----------+----------+------------+
+
+
+.. csv-table:: PXC Section
+  :header: "Key", "Value", "Example", "Description"
+  :widths: 35, 8,20,25
+
+  "size", "int", ``3``, The size of the Percona XtraDB cluster must be >= 3 for `High Availability <https://www.percona.com/doc/percona-xtradb-cluster/5.7/intro.html>`_
+  "allowUnsafeConfigurations", "string",``false``, "Prevents users from configuring a cluster with unsafe parameters such as starting the cluster with less than 3 nodes or starting the cluster without TLS/SSL certificates"
+  image, string, ``percona/percona-xtradb-cluster-operator:1.0.0-pxc``, The Docker image of the Percona cluster used.
+  readinessDelaySec, int, ``15``, Adds a delay before a run check to verify the application is ready to process traffic
+  livenessDelaySec, int, ``300``, Adds a delay before the run check, which ensures the application is healthy and capable of processing requests
+  forceUnsafeBootstrap, string, ``false``, Prevents the use of outdated and unsafe settings. The setting can be reset in case of a sudden crash, when all nodes may be considered unsafe to bootstrap from. The setting lets a node to be selected, set `safe_to_bootstrap`, and provides data recovery.
```

Comment:
this option about Bootstrap, not about "outdated and unsafe settings"
```suggestion
  forceUnsafeBootstrap, string, ``false``, The setting can be reset in case of a sudden crash, when all nodes may be considered unsafe to bootstrap from. The setting lets a node to be selected, set `safe_to_bootstrap`, and provides data recovery.
```

Reply:
Removed string.